src/main/java/dev/jbang/source/ScriptSource.java

```diff
@@ -506,6 +506,11 @@ private void collectAllSources(Set<ResourceRef> refs, List<ScriptSource> scripts
 		return Stream.concat(func.apply(this).stream(), subs).collect(Collectors.toList());
 	}
 
+	@Override
+	public boolean isCreatedJar() {
```

Comment:
Not sure if "created" is completely correct. It could already exist from a previous build. It just exists and from that you _could_ infer that it has been built because otherwise we'd have taken the JarSource code path.
Also, the fact that JarSource returns false feels counter-intuitive.

Reply:
Yes the name sucks. Not sure what to use for "created just now" :)

If there is a existing jar it becomes a jarsource hence why it returns false. 

The use is for CDs where the jdk stupidly enough can't figure out to handle by itself if the jar is preexisting or not.